src/js/form.js

```diff
@@ -0,0 +1,124 @@
+import intlTelInput from 'intl-tel-input/index.js';
+import 'intl-tel-input/build/js/utils';
+import 'flatpickr';
+import rangePlugin from '../../node_modules/flatpickr/dist/plugins/rangePlugin';
```

Comment:
Have you tried to get this plugin by absolute path? `flatpickr/dist/plugins/rangePlugin`